homeassistant/components/updater.py

```diff
@@ -97,9 +97,15 @@ def check_new_version(now):
 
         newest, releasenotes = result
 
+        # Skip on dev
         if newest is None or 'dev' in current_version:
             return
 
+        # Load data from supervisor on hass.io
+        if hass.components.hassio.is_hassio():
+            newest = hass.components.hassio.get_homeassistant_version()
```

Comment:
I'm confused why this is necessary. If we're running Home Assistant, then this is not necessary.

Reply:
nevermind, this is not theversion that is running but the version that is available